VisualStudio.gitignore

```diff
@@ -148,9 +148,11 @@ publish/
 # Publish Web Output
 *.[Pp]ublish.xml
 *.azurePubxml
-# TODO: Comment the next line if you want to checkin your web deploy settings
-# but database connection strings (with potential passwords) will be unencrypted
-*.pubxml
+# TODO: Uncomment the next line to ignore your web deploy settings.
+# By default, sensitive information, such as encrypted password
+# should be stored in the .pubxml.user file.
+#*.pubxml
+*.pubxml.user
```

Comment:
It's not clear to me why this rule isn't covered by the `*.user` on line 8.

Reply:
So this PR isn't necessary then, right?

Reply:
Correct.